lib/matplotlib/tests/test_mathtext.py

```diff
@@ -453,10 +452,10 @@ def test_mathtext_fallback(fallback, fontlist):
     fig.savefig(buff, format="svg")
     tspans = (ET.fromstring(buff.getvalue())
               .findall(".//{http://www.w3.org/2000/svg}tspan[@style]"))
-    # Getting the last element of the style attrib is a close enough
-    # approximation for parsing the font property.
-    char_fonts = [shlex.split(tspan.attrib["style"])[-1] for tspan in tspans]
-    assert char_fonts == fontlist
+    char_fonts = [
+        re.search(r"font-family: '([\w ]+)';?", tspan.attrib["style"]).group(1)
```

Comment:
```suggestion
        re.search(r"font-family: '([\w ]+)'", tspan.attrib["style"]).group(1)
```
If `;` is optional at the end, we can simply leave it out of the regexp.